Cargo.lock

```diff
@@ -6279,6 +6279,15 @@ dependencies = [
  "hashbrown 0.13.2",
 ]
 
+[[package]]
+name = "lru"
+version = "0.12.3"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "d3262e75e648fce39813cb56ac41f3c3e3f65217ebf3844d818d1f9398cfb0dc"
+dependencies = [
+ "hashbrown 0.14.1",
+]
+
```

Comment:
Why is this needed?

Reply:
hashbrowns? Needed because they're delicious. Not sure how this got in here, though. Removing...